packages/cli/package.json

```diff
@@ -9,8 +9,8 @@
   "dependencies": {
     "@stoplight/http-spec": "^4.0.0",
     "@stoplight/prism-core": "^4.1.3",
```

Comment:
@P0lip :wave: AFAIR every time a new prism version has been shipped, all the packages were updated to expose the same version defined "lerna version" (cf. https://github.com/stoplightio/prism/pull/1562/files for instance).

Here, prism-core is left unmodified at version 4.1.3. Is that expected?

Reply:
It could be there has been no change made to prism-core, and that's the reason?
I'm not sure, I'll let my fellows from @stoplightio/platinum-falcons answer, as they have more context here than I do.

Reply:
Yep `learn version` only bumps the version if the package has changed: https://github.com/lerna/lerna/tree/main/commands/version#lifecycle-scripts